pkg/collector/osmlogs_collector.go

```diff
@@ -62,7 +63,8 @@ func (collector *OsmLogsCollector) Collect() error {
 	for _, meshName := range meshList {
 		namespacesInMesh, err := getResourceList("namespaces", "openservicemesh.io/monitored-by="+meshName, "-o=jsonpath={..name}", " ")
 		if err != nil {
-			return err
```

Comment:
But this should return err if `getResourceList()` err's because of 
`	outputStreams, err := utils.RunCommandOnContainerWithOutputStreams("kubectl", "get", resource, "--all-namespaces", "--selector", label, outputFormat)`